src/dispatch/incident_type/models.py

```diff
@@ -15,6 +15,7 @@ class IncidentType(Base):
     slug = Column(String)
     description = Column(String)
     visibility = Column(String, default=Visibility.open)
+    plugin_metadata = Column(JSON)
```

Comment:
If we set the default to be an empty dict via `plugin_metadata = db.Column(JSON, nullable=False, default=dict, server_default='{}')` I think we can remove a lot of checks for `None` and returning empty dicts.

Reply:
updated